test/unit/css.js

```diff
@@ -1401,6 +1401,20 @@ testIframe(
 	}
 );
 
+testIframe(
+	"Test computeStyleTests for hidden iframe",
+	"css/cssComputeStyleTests.html",
+	function( assert, jQuery, window, document, initialHeight ) {
+		assert.expect( 2 );
+		assert.strictEqual( initialHeight, 0, "initial height should be undefined" );
```

Comment:
Looks like Safari actually is reporting offsetHeight for the div in the hidden iframe: https://github.com/jquery/jquery/actions/runs/6634909917/job/18189746921?pr=5317#step:9:161
```suggestion
		assert.strictEqual( initialHeight === 0 ? 20 : initialHeight, 20,
			"hidden-frame content sizes should be zero or accurate" );
```